benchmark/README.md

```diff
@@ -23,97 +24,96 @@ Accordingly, in terms of client instrumentation performance I suggest the follow
 
 ## Benchmark Results
 
-These benchmarks were run using JMH on a 2-core MacBook Pro with a 2.5GHz i5 processor, 
-with Oracle Java 64 1.7.0\_51.
+These benchmarks were run using JMH on a Linux laptop with a 4 Core Intel i7-8550U CPU with OpenJDK 1.8.0_292-b10.
 
 ### Counters
     java -jar target/benchmarks.jar CounterBenchmark -wi 5 -i 5 -f 1 -t 1
-    i.p.b.CounterBenchmark.codahaleCounterIncBenchmark                    avgt        5   11.554 ± 0.251  ns/op
-    i.p.b.CounterBenchmark.codahaleMeterMarkBenchmark                     avgt        5   75.305 ± 7.147  ns/op
-    i.p.b.CounterBenchmark.prometheusCounterChildIncBenchmark             avgt        5   13.249 ± 0.029  ns/op
-    i.p.b.CounterBenchmark.prometheusCounterIncBenchmark                  avgt        5  127.397 ± 4.072  ns/op
-    i.p.b.CounterBenchmark.prometheusSimpleCounterChildIncBenchmark       avgt        5   12.989 ± 0.285  ns/op
-    i.p.b.CounterBenchmark.prometheusSimpleCounterIncBenchmark            avgt        5   54.822 ± 7.994  ns/op
-    i.p.b.CounterBenchmark.prometheusSimpleCounterNoLabelsIncBenchmark    avgt        5   13.131 ± 1.661  ns/op
+    i.p.c.b.CounterBenchmark.codahaleCounterIncBenchmark                    avgt        5   8.063 ± 0.440  ns/op
+    i.p.c.b.CounterBenchmark.codahaleMeterMarkBenchmark                     avgt        5  38.625 ± 2.111  ns/op
+    i.p.c.b.CounterBenchmark.prometheusCounterChildIncBenchmark             avgt        5   9.869 ± 0.252  ns/op
```

Comment:
Incidentally, we can remove these old Java client benchmarks. They've not been relevant in many years.

Reply:
Do you mean remove from `README.md`, or remove the implementations as well? Are there benchmarks that are still relevant apart from the `ExemplarsBenchmark`?

Reply:
The implementations for the old java client. The benchmarks are still relevant, that client isn't however.

Reply:
Ok, I removed the old client from the benchmarks.